lib/SIL/Verifier/MemoryLifetimeVerifier.cpp

```diff
@@ -788,7 +788,9 @@ void MemoryLifetimeVerifier::checkBlock(SILBasicBlock *block, Bits &bits) {
           requireBitsSet(bits, sbi->getDest(), &I);
           locations.clearBits(bits, sbi->getDest());
         } else if (auto *lbi = dyn_cast<LoadBorrowInst>(ebi->getOperand())) {
-          requireBitsSet(bits, lbi->getOperand(), &I);
+          if (!lbi->isUnchecked()) {
+            requireBitsSet(bits, lbi->getOperand(), &I);
+          }
```

Comment:
May need a similar bailout at - https://github.com/swiftlang/swift/blob/11696cd452ae043223155dd1935c571533f79c9a/lib/SIL/Verifier/MemoryLifetimeVerifier.cpp#L792

Reply:
Thanks for taking a look! The place you linked is the same place I already edited. Did you have somewhere else in mind too?

Reply:
Sorry, that was it!